content/en/docs/tasks/tools/install-minikube.md

```diff
@@ -208,54 +93,18 @@ To install Minikube manually on Windows, download [`minikube-windows-amd64`](htt
 
 {{% /capture %}}
 
-## Confirm Installation
-
-To confirm successful installation of both a hypervisor and Minikube, you can run the following command to start up a local Kubernetes cluster:
-
-{{< note >}}
+## Starting your first cluster
 
-For setting the `--driver` with `minikube start`, enter the name of the hypervisor you installed in lowercase letters where `<driver_name>` is mentioned below. A full list of `--driver` values is available in [specifying the VM driver documentation](https://kubernetes.io/docs/setup/learning-environment/minikube/#specifying-the-vm-driver).
-
-{{< /note >}}
+Run:
 
-```shell
-minikube start --driver=<driver_name>
-```
-
-Once `minikube start` finishes, run the command below to check the status of the cluster:
-
-```shell
-minikube status
-```
-
-If your cluster is running, the output from `minikube status` should be similar to:
-
-```
-host: Running
-kubelet: Running
-apiserver: Running
-kubeconfig: Configured
-```
-
-After you have confirmed whether Minikube is working with your chosen hypervisor, you can continue to use Minikube or you can stop your cluster. To stop your cluster, run:
-
-```shell
-minikube stop
-```
-
-## Clean up local state {#cleanup-local-state}
-
-If you have previously installed Minikube, and run:
 ```shell
 minikube start
 ```
 
-and `minikube start` returned an error:
-```
-machine does not exist
-```
+minikube will automatically select an appropriate container or hypervisor driver, but will mention alternatives it available. You may select an alternative driver by using the `--driver` flag. For more information, see the [minikube drivers documentation](https://minikube.sigs.k8s.io/docs/drivers/)
```

Comment:
Possibly reword `alternatives it`:
During startup, minikube selects an appropriate container or hypervisor driver and informs you
about the availability of alternative containers or drivers.

Reply:
nit: Add a period at end of sentence
For more information, see the [minikube drivers documentation](https://minikube.sigs.k8s.io/docs/drivers/).

What do you think about moving, `If you need help ...` to the What's next section?

Reply:
Great suggestions. How's this look?